rules/aws/persistence_ec2_security_group_configuration_change_detection.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/05/05"
+maturity = "development"
+updated_date = "2021/05/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Detects threat actors making configuration change to EC2 security groups. Threat actors may be able to maintain
+persistence, exfiltrate, and pivot in a AWS environment.
```

Comment:
May want to change how this is written a bit. May be over assumptive to stat that a threat actor is performing the action. If you use more passive verbiage, it is more accurate (and forgiving when FP). Most of the cloud rules have a good structure which you can reference.